src/main/groovy/io/seqera/wave/controller/ContainerTokenController.groovy

```diff
@@ -244,6 +244,7 @@ class ContainerTokenController {
                 scanId,
                 ip,
                 offset)
+                .withAccessToken(req.towerAccessToken)
```

Comment:
Why this is needed?

Reply:
This is needed to get the org information from the license manager

Reply:
no needed, there platform identity
I will remove it